libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/model/SpeechVolume.kt

```diff
@@ -0,0 +1,12 @@
+package com.mapbox.navigation.ui.voice.model
+
+import androidx.annotation.FloatRange
+
+/**
+ * The state is returned if we change the speech volume.
+ * @param level
+ */
+class SpeechVolume(
```

Comment:
Should this be a builder? 🤔 

We can also explore @cafesilencio's idea 

```kotlin
interface SomeValue {
    fun getValue(): String?
}
interface SomeValueError: SomeValue {
    fun getThrowable(): Throwable
}
interface SomeVolumeValue: SomeValue {
    fun getVolumeValue: SomeType
}
```

then in the future if there is something added

```kotlin
interface someValueV2: SomeValue {
    fun getAnotherValue(): Whatever
}
```

backwards compatible while also extensible.